dev-infrastructure/region-pipeline.yaml

```diff
@@ -0,0 +1,16 @@
+serviceGroup: Microsoft.Azure.ARO.Test
+rolloutName: Region Rollout
+resourceGroups:
+- name: {{ .regionRG }}
+  subscription: {{ .serviceClusterSubscription }}
+  steps:
+  - name: region
+    action: ARM
+    template: templates/region.bicep
+    parameters: configurations/region.tmpl.bicepparam
+  - name: region
```

Comment:
@whober0521 good to validate duplicate names and that `dependsOn` actually refers to a named step before this one.

Another question - if these steps are in a list, and they are executed one after the other, is the `dependsOn` actually necessary?

Reply:
i've added some basic validation that the referenced step exists, but needs more love (circular deps etc)